core/controlplane/config/config.go

```diff
@@ -750,8 +755,20 @@ type KubeResourcesAutosave struct {
 }
 
 type CloudWatchLogging struct {
-	Enabled         bool `yaml:"enabled"`
-	RetentionInDays int  `yaml:"retentionInDays"`
+	Enabled          bool `yaml:"enabled"`
+	RetentionInDays  int  `yaml:"retentionInDays"`
+	RealtimeFeedback `yaml:"realtimeFeedback"`
```

Comment:
I'm not very sure but perhaps `LocalStreaming` would better express its intention?